code/modules/clothing/outfits/standard.dm

```diff
@@ -252,6 +252,7 @@
 	uniform = /obj/item/clothing/under/color/lightpurple
 	suit = /obj/item/clothing/suit/wizrobe
 	shoes = /obj/item/clothing/shoes/sandal/magic
+	digitigrade_shoes = /obj/item/clothing/shoes/xeno_wraps/magic
```

Comment:
there isn't ever going to be a situation where this is used unless you forcefully equip a lizard with this outfit since wizard antag sets the owner to human. if shards are really such a problem then make the wooden sandals wearable by all (it makes sense)

Reply:
Shoes don't have a system for alternate digitigrade sprites like suits or jumpsuits do (and I don't really feel like coding that in at the moment, maybe in another PR) so it would look weird, instead I just made a separate kind of footwraps for wizards in the clothing crate because it's a simple change that doesn't harm anyone. Also I don't see any reason NOT to add it to the outfit, it's mainly just for the sake of consistency (and also because I want to make wizard not force you to be human in a future PR)